src/dapi/sensor.ts

```
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+export async function requestEnlistSensor(
+  metadata: string,
+  stakeAmount: string,
+  price: string
+) {
+  try {
+    const response = await axios.post(`/sensorregistry/enlist`, {
+      _metadata: metadata,
+      _stakeAmount: stakeAmount,
+      _price: price,
+    });
+    return response.data.uuid;
+  } catch (error) {
+    console.error('Failed to request sensor enlisting with error', error);
```

REQ: think about this -> you enlist a sensor, you wrap it in a try catch, and on error you log.

You are failing silently here, if it's what you want, then tell me by adding a comment.
In this case it might even be the case but I don't know so now I'm thinking you didn't do it on purpose...

---

It was indeed something I've looked over